### PR TITLE
Lazarus Involved Stake.com Article

### PR DESCRIPTION
**Pull request by JediFaust**
Stake.com incident Lazarus Group involved

### CODE REVIEW

File: content/attacks/posts/2023-09-04-Stake.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  - Gambling Platform
+attack-types:
+  - Wallet Hack
+  - Infrastructure Attack
+title: "Stake.com Suffers $41.4 Million Security Breach"
+---
+
+## Summary
+
+On September 4, 2023, Stake.com, a crypto gambling protocol offering casino games and sports betting, was targeted by the Lazarus Group (also known as APT38), a group of DPRK cyber actors. The group exploited access control vulnerabilities and extracted approximately $41.4 million worth of various digital assets from the platform's hot wallets across Ethereum, Binance Smart Chain (BSC), and Polygon networks. Stake.com reassured users that their funds were safe, and all other wallets remained operational. The stolen funds were accurately distributed across multiple addresses and chains. Interestingly, some affected addresses were holding over $13 million worth of various tokens as of September 7, 2023.
```
Comment (jhirschkorn) [inline content/attacks/posts/2023-09-04-Stake.md:15]: Last sentence:
Some affected addresses held over $13 million worth of various tokens as of...

File: content/attacks/posts/2023-09-04-Stake.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+## Attackers
+
+The FBI has identified the Lazarus Group as the responsible party for this attack. The following addresses were used to transfer the funds:
```
Comment (jhirschkorn) [inline content/attacks/posts/2023-09-04-Stake.md:19]: Can you add a link to the FBI source for this?

**Review by jhirschkorn — CHANGES_REQUESTED** [review]

File: content/attacks/posts/2023-09-04-Stake.md (hunk 1 above)
Comment (JediFaust) [reply content/attacks/posts/2023-09-04-Stake.md]: Done

**Review by jhirschkorn — APPROVED** [review]

**Pull request merged**